libraries/chain/db_debug.cpp

```diff
@@ -93,21 +93,10 @@ void database::debug_dump(const genesis_state_type& genesis_state)
 //      edump((total_balances[asset_obj.id])(asset_obj.dynamic_asset_data_id(db).current_supply ) );
    }
 
-   // Initial balances
-   for( const auto& ib : genesis_state.initial_balances )
-   {
-      total_balances[asset_id_type()] += ib.amount;
-   }
-
-   // Initial vesting balances
-   for( const auto& iv : genesis_state.initial_vesting_balances )
-   {
-      total_balances[asset_id_type()] += iv.amount;
-   }
-
    if( total_balances[asset_id_type()].value != core_asset_data.current_supply.value )
    {
       edump( (total_balances[asset_id_type()].value)(core_asset_data.current_supply.value ));
+      exit(0);
```

Comment:
Don't call exit(). Better throw. (And if you do call exit, use a non-zero exit value to indicate an error. But don't call it.)

Reply:
changed it, thanks.